unit2-java/lesson10.rst

```diff
@@ -5,7 +5,19 @@ Goals
 -----
 
 - Java development and tooling; exceptions
-   - Gradle, IDE in detail, compilation and JVM
+- Gradle, IDE in detail, compilation and JVM
 
 Objectives
 ----------
+
+- Describe what an IDE like IntelliJ offers programmers that text
```

Comment:
Formatting edit: Check your indentation.  All of the words before "editors" are bold-italic.

Also, this objective is a little out of place, since the students have been using the IDE for several lessons and have explored some of its shortcuts.  How about tying this objective to what the IDE offers in terms of creating a web app?

Reply:
You're correct on placement. As discussed in the meeting yesterday, we should probably bump these objectives up. I moved them here (from lesson 8) just to get them out of the way for now.